src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -135,7 +135,8 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 CancellationToken cancellationToken)
             {
                 parameter = (IParameterSymbol)semanticModel.GetDeclaredSymbol(parameterNode, cancellationToken);
```

Comment:
we likely have this issue in a lot of places :(

tagging @dpoeschl  for visibility in his 'add parameter' work.

Reply:
Interesting that the [file](http://sourceroslyn.io/#Microsoft.CodeAnalysis/Compilation/Extensions.cs,136) with GetDeclaredSymbol is nullable enabled, the comment on the method says it can return null, but return type is not annotated as a nullable symbol. Should we file a compiler bug to catch these issues?

Reply:
Opened up an issue here: #42992